lib/src/rules/prefer_relative_imports.dart

```diff
@@ -85,6 +66,7 @@ class _Visitor extends SimpleAstVisitor<void> {
     String importUri = node?.uri?.stringValue;
```

Comment:
If this method is only invoked from `visitImportDirective`, then `node` can't be `null` so there's no need for the null check. Consider using `node.uriContent`.

Reply:
Thanks!